corehq/messaging/scheduling/forms.py

```diff
@@ -2683,6 +2711,9 @@ class ConditionalAlertScheduleForm(ScheduleForm):
     START_OFFSET_NEGATIVE = 'NEGATIVE'
     START_OFFSET_POSITIVE = 'POSITIVE'
 
+    MIN_SMS_STALE_INTERVAL = 0  # hours
```

Comment:
How about just renaming this to `MIN_SMS_STALE_HOURS`, so you don't have to come here and read the comment?